optimize/backend/src/main/java/io/camunda/optimize/rest/cloud/CCSaaSNotificationClient.java

```diff
@@ -82,8 +82,8 @@ public synchronized void refreshAccessTokenIfRequired() {
     }
   }
 
```

Comment:
the code below was being detected as a bug, because the `synchronized` word was missing. It is my understanding that this was a bug for real, so I added the keyword in the places where it was detected as missing. From what I could see, nothing will change

Reply:
I thought the critical section was the access to the `accessToken`, so I thought that the bug was legit.
I'll check again and revert this, in case.

I am assuming it is possible to override or ignore the error I'm getting?